app/views/downloads/download_linux.html.erb

```diff
@@ -12,49 +12,83 @@
 
   <h3>Debian/Ubuntu</h3>
   <p>For the latest stable version for your release of Debian/Ubuntu</p>
-  <code># apt-get install git</code>
+  <pre># apt-get install git</pre>
   <p>For Ubuntu, this PPA provides the latest stable upstream Git version</p>
-  <code># add-apt-repository ppa:git-core/ppa</code>
-  <code># apt update; apt install git</code>
+  <pre># add-apt-repository ppa:git-core/ppa</pre>
+  <pre># apt update; apt install git</pre>
+  <p>Optionally, if you would also like to install gitk, then run the following:</p>
+  <pre># sudo apt install gitk</pre>
   
   <h3>Fedora</h3>
-  <code># yum install git</code> (up to Fedora 21)<br>
-  <code># dnf install git</code> (Fedora 22 and later)
+  <pre># yum install git</pre> (up to Fedora 21)<br>
+  <pre># dnf install git</pre> (Fedora 22 and later)
+  <p>Optionally, if you would also like to install gitk, then run the following:</p>
+  <pre># sudo yum install tk</pre>
+  <pre># sudo yum install gitk</pre>
```

Comment:
Do we need two lines for this? Can `yum` not figure out the dependencies automatically? If not, can we provide both packages on a single command-line (it's been ages since I've had to use it, so the answer might well be "no and no").

Reply:
I did some researching on this and some people said that they had to run:
```bash
sudo yum install tk
```
After they ran:
```bash
sudo yum install gitk
```
For gitk to actually work. I am not on Fedora, I am on Ubuntu, so I can't confirm that. It is best to just make them do that just in case.

Reply:
OK. Let's leave it like that for now, then, and if somebody with practical knowledge shows up they can suggest a change.